07-01-2025 21:26:04.688 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 17.0.5 on NguyenLoc with PID 1896 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:26:04.690 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:26:04.691 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:26:28.617 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 17.0.5 on NguyenLoc with PID 17864 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:26:28.619 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:26:28.619 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:26:58.597 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 17.0.5 on NguyenLoc with PID 5600 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:26:58.599 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:26:58.600 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:27:00.349 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:27:00.816 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:01.123 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:01.437 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:01.749 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:02.062 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:02.375 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:02.689 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:03.003 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:03.310 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:03.621 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:03.933 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:04.234 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:04.544 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:04.845 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:05.159 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:05.472 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:05.779 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:06.093 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:06.405 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:06.719 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:07.030 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:27:07.342 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:467)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:595)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:382)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$dfdbc163.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:547)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:636)
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:158)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.createNativeEntityManager(AbstractEntityManagerFactoryBean.java:585)
	at jdk.internal.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.invokeProxyMethod(AbstractEntityManagerFactoryBean.java:487)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean$ManagedEntityManagerFactoryInvocationHandler.invoke(AbstractEntityManagerFactoryBean.java:734)
	at jdk.proxy2/jdk.proxy2.$Proxy82.createNativeEntityManager(Unknown Source)
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:485)
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:410)
	... 11 common frames omitted
07-01-2025 21:31:45.946 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 15948 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:31:45.949 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:31:45.949 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:31:48.087 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:31:49.051 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 3.543 seconds (JVM running for 6.431)
07-01-2025 21:33:37.754 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, customerId=28169d2}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:33:37.763 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:33:37.789 INFO  --- com.kaiasia.app.ValidateImpl             - 703 Enquiry not exist
07-01-2025 21:34:43.769 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 12360 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:34:43.772 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:34:43.772 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:34:45.572 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:34:46.447 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 3.029 seconds (JVM running for 4.082)
07-01-2025 21:35:19.068 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, customerId=28169d2}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:35:19.075 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:35:19.097 INFO  --- com.kaiasia.app.ValidateImpl             - 706 MISSING FIELD#userName
07-01-2025 21:35:55.574 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:35:55.574 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:35:55.574 INFO  --- com.kaiasia.app.ValidateImpl             - 706 MISSING FIELD#password
07-01-2025 21:36:09.889 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:36:09.889 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:36:09.890 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:36:09.890 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260569890000_0000_822bf968a6ed4ce690faec706ab75772#[API-BEGIN]
07-01-2025 21:36:09.899 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260569890000_0000_822bf968a6ed4ce690faec706ab75772#BEGIN
07-01-2025 21:36:09.912 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260569890000_0000_822bf968a6ed4ce690faec706ab75772#END
07-01-2025 21:36:09.913 INFO  --- com.kaiasia.app.core.ProcessService      - 1736260569890000_0000_822bf968a6ed4ce690faec706ab75772##pushApiReqToQueue#Duration:23
07-01-2025 21:37:09.994 ERROR --- com.kaiasia.app.core.ProcessService      - 1736260569890000_0000_822bf968a6ed4ce690faec706ab75772#Return timeout, because client get response at time over timeout value
07-01-2025 21:37:09.994 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260569890000_0000_822bf968a6ed4ce690faec706ab75772##END
07-01-2025 21:37:09.995 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260569890000_0000_822bf968a6ed4ce690faec706ab75772#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=998, desc=Timeout {0}))#Total time: 60106
07-01-2025 21:37:37.206 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:37:37.206 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:37:37.206 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:37:37.206 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c#[API-BEGIN]
07-01-2025 21:37:37.207 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c#BEGIN
07-01-2025 21:37:37.208 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c#END
07-01-2025 21:37:37.210 INFO  --- com.kaiasia.app.core.ProcessService      - 1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c##pushApiReqToQueue#Duration:4
07-01-2025 21:38:37.285 ERROR --- com.kaiasia.app.core.ProcessService      - 1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c#Return timeout, because client get response at time over timeout value
07-01-2025 21:38:37.286 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c##END
07-01-2025 21:38:37.286 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260657206000_0000_3b29c65923504bd1ad4a502efafdb89c#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=998, desc=Timeout {0}))#Total time: 60080
07-01-2025 21:39:06.241 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.TransactionSystemException: Could not roll back JPA transaction; nested exception is org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.springframework.orm.jpa.JpaTransactionManager.doRollback(JpaTransactionManager.java:593)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processRollback(AbstractPlatformTransactionManager.java:835)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.rollback(AbstractPlatformTransactionManager.java:809)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.completeTransactionAfterThrowing(TransactionAspectSupport.java:672)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:392)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$60183125.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:127)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304)
	at org.hibernate.engine.transaction.internal.TransactionImpl.rollback(TransactionImpl.java:142)
	at org.springframework.orm.jpa.JpaTransactionManager.doRollback(JpaTransactionManager.java:589)
	... 11 common frames omitted
Caused by: java.sql.SQLException: Connection is closed
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection.lambda$getClosedConnection$0(ProxyConnection.java:515)
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection$$Lambda$585/195359600.invoke(Unknown Source)
	at com.sun.proxy.$Proxy81.rollback(Unknown Source)
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396)
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java)
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121)
	... 14 common frames omitted
07-01-2025 21:39:06.546 ERROR --- com.kaiasia.app.core.job.ApiFetchReqTask - EXCEPTION FETCH DB: {}
org.springframework.transaction.TransactionSystemException: Could not roll back JPA transaction; nested exception is org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.springframework.orm.jpa.JpaTransactionManager.doRollback(JpaTransactionManager.java:593)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processRollback(AbstractPlatformTransactionManager.java:835)
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.rollback(AbstractPlatformTransactionManager.java:809)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.completeTransactionAfterThrowing(TransactionAspectSupport.java:672)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:392)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.kaiasia.app.core.dao.APIDaoManager$$EnhancerBySpringCGLIB$$60183125.fetchAPIReqs(<generated>)
	at com.kaiasia.app.core.job.ApiFetchReqTask.run(ApiFetchReqTask.java:40)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:127)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304)
	at org.hibernate.engine.transaction.internal.TransactionImpl.rollback(TransactionImpl.java:142)
	at org.springframework.orm.jpa.JpaTransactionManager.doRollback(JpaTransactionManager.java:589)
	... 11 common frames omitted
Caused by: java.sql.SQLException: Connection is closed
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection.lambda$getClosedConnection$0(ProxyConnection.java:515)
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection$$Lambda$585/195359600.invoke(Unknown Source)
	at com.sun.proxy.$Proxy81.rollback(Unknown Source)
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396)
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java)
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121)
	... 14 common frames omitted
07-01-2025 21:39:30.950 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:39:30.950 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:39:30.950 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:39:30.950 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6#[API-BEGIN]
07-01-2025 21:39:30.950 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6#BEGIN
07-01-2025 21:39:30.953 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6#END
07-01-2025 21:39:30.956 INFO  --- com.kaiasia.app.core.ProcessService      - 1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6##pushApiReqToQueue#Duration:6
07-01-2025 21:39:31.569 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:39:31.569 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:39:31.569 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:39:31.569 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:39:58.038 ERROR --- com.kaiasia.app.register.ServiceRegister - {}
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.kaiasia.app.register.RequestProcessor.execute(RequestProcessor.java:20)
	at com.kaiasia.app.register.ServiceRegister.processAuthenType(ServiceRegister.java:81)
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:35)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `ms.apiclient.model.ApiHeader` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('eyJyZXFUeXBlIjoiUkVRVUVTVCIsImFwaSI6IkF1dGhfQVBJIiwiYXBpS2V5IjoiYXV0aHE1MWtsZm9uaTFlenhsNWYyY2twZngyNDgiLCJjaGFubmVsIjoiQVBJIiwibG9jYXRpb24iOiJQQy9JT1MiLCJyZXF1ZXN0QVBJIjoiRkUgQVBJIiwicmVxdWVzdE5vZGUiOiJub2RlIDAxIiwiZHVyYXRpb24iOm51bGwsInByaW9yaXR5IjoxLCJjb250ZXh0IjpudWxsLCJzeW5hc3luIjoiZmFsc2UifQ==')
 at [Source: (String)""eyJyZXFUeXBlIjoiUkVRVUVTVCIsImFwaSI6IkF1dGhfQVBJIiwiYXBpS2V5IjoiYXV0aHE1MWtsZm9uaTFlenhsNWYyY2twZngyNDgiLCJjaGFubmVsIjoiQVBJIiwibG9jYXRpb24iOiJQQy9JT1MiLCJyZXF1ZXN0QVBJIjoiRkUgQVBJIiwicmVxdWVzdE5vZGUiOiJub2RlIDAxIiwiZHVyYXRpb24iOm51bGwsInByaW9yaXR5IjoxLCJjb250ZXh0IjpudWxsLCJzeW5hc3luIjoiZmFsc2UifQ==""; line: 1, column: 1]
	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:63)
	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1733)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1358)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._deserializeFromString(StdDeserializer.java:311)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromString(BeanDeserializerBase.java:1500)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:197)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:187)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4730)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3677)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3645)
	at com.kaiasia.app.service.Auth_api.utils.ApiUtils.buildApiHeader(ApiUtils.java:11)
	at com.kaiasia.app.service.Auth_api.api.login.LoginService.process(LoginService.java:96)
	... 13 common frames omitted
07-01-2025 21:39:58.039 ERROR --- com.kaiasia.app.ProcessImpl              - {}
java.lang.NullPointerException: null
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:38)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
07-01-2025 21:39:58.041 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6#saveResponse#Duration: 2
07-01-2025 21:39:58.252 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6##END
07-01-2025 21:39:58.252 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260770950000_0000_29d2020e4b784199a16fd89c1a9436a6#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=27039, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=999, desc=Internal Server Error java.lang.NullPointerException))#Total time: 27303
07-01-2025 21:40:25.549 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 21428 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:40:25.553 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:40:25.553 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:40:27.398 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:40:28.204 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 3.036 seconds (JVM running for 4.081)
07-01-2025 21:40:31.102 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:40:31.109 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:40:31.133 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:40:31.134 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786#[API-BEGIN]
07-01-2025 21:40:31.156 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786#BEGIN
07-01-2025 21:40:31.167 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786#END
07-01-2025 21:40:31.168 INFO  --- com.kaiasia.app.core.ProcessService      - 1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786##pushApiReqToQueue#Duration:34
07-01-2025 21:40:31.565 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:40:31.565 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:40:31.565 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:40:31.566 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:40:32.507 ERROR --- com.kaiasia.app.register.ServiceRegister - {}
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.kaiasia.app.register.RequestProcessor.execute(RequestProcessor.java:20)
	at com.kaiasia.app.register.ServiceRegister.processAuthenType(ServiceRegister.java:81)
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:35)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: Cannot construct instance of `ms.apiclient.model.ApiHeader` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('"eyJyZXFUeXBlIjoiUkVRVUVTVCIsImFwaSI6IkF1dGhfQVBJIiwiYXBpS2V5IjoiYXV0aHE1MWtsZm9uaTFlenhsNWYyY2twZngyNDgiLCJjaGFubmVsIjoiQVBJIiwibG9jYXRpb24iOiJQQy9JT1MiLCJyZXF1ZXN0QVBJIjoiRkUgQVBJIiwicmVxdWVzdE5vZGUiOiJub2RlIDAxIiwiZHVyYXRpb24iOm51bGwsInByaW9yaXR5IjoxLCJjb250ZXh0IjpudWxsLCJzeW5hc3luIjoiZmFsc2UifQ=="')
 at [Source: UNKNOWN; byte offset: #UNKNOWN]
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4449)
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4380)
	at com.kaiasia.app.service.Auth_api.utils.ApiUtils.buildApiHeader(ApiUtils.java:11)
	at com.kaiasia.app.service.Auth_api.api.login.LoginService.process(LoginService.java:96)
	... 13 common frames omitted
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `ms.apiclient.model.ApiHeader` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('"eyJyZXFUeXBlIjoiUkVRVUVTVCIsImFwaSI6IkF1dGhfQVBJIiwiYXBpS2V5IjoiYXV0aHE1MWtsZm9uaTFlenhsNWYyY2twZngyNDgiLCJjaGFubmVsIjoiQVBJIiwibG9jYXRpb24iOiJQQy9JT1MiLCJyZXF1ZXN0QVBJIjoiRkUgQVBJIiwicmVxdWVzdE5vZGUiOiJub2RlIDAxIiwiZHVyYXRpb24iOm51bGwsInByaW9yaXR5IjoxLCJjb250ZXh0IjpudWxsLCJzeW5hc3luIjoiZmFsc2UifQ=="')
 at [Source: UNKNOWN; byte offset: #UNKNOWN]
	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:63)
	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1733)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1358)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._deserializeFromString(StdDeserializer.java:311)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromString(BeanDeserializerBase.java:1500)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:197)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:187)
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4444)
	... 16 common frames omitted
07-01-2025 21:40:32.508 ERROR --- com.kaiasia.app.ProcessImpl              - {}
java.lang.NullPointerException: null
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:38)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
07-01-2025 21:40:32.511 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786#saveResponse#Duration: 3
07-01-2025 21:40:32.708 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786##END
07-01-2025 21:40:32.708 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260831133000_0000_9b76b604c91248c3ab5ceefe155d5786#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=1508, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=999, desc=Internal Server Error java.lang.NullPointerException))#Total time: 1606
07-01-2025 21:43:00.497 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 15200 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:43:00.499 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:43:00.500 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:43:02.262 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:43:03.204 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 3.069 seconds (JVM running for 4.094)
07-01-2025 21:43:06.741 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:43:06.750 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:43:06.771 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:43:06.771 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c#[API-BEGIN]
07-01-2025 21:43:06.795 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c#BEGIN
07-01-2025 21:43:06.802 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c#END
07-01-2025 21:43:06.803 INFO  --- com.kaiasia.app.core.ProcessService      - 1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c##pushApiReqToQueue#Duration:32
07-01-2025 21:43:06.899 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:43:06.899 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:43:06.899 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:43:06.899 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:43:10.273 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:43:31.383 ERROR --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#while calling api T24_UTIL_API: I/O error on POST request for "http://14.225.254.212:8082/T24_UTIL_API/process": Connection timed out: connect; nested exception is java.net.ConnectException: Connection timed out: connect
07-01-2025 21:43:31.386 ERROR --- ms.apiclient.client.common.ApiCallClient - {}#exception {}
org.springframework.web.client.ResourceAccessException: I/O error on POST request for "http://14.225.254.212:8082/T24_UTIL_API/process": Connection timed out: connect; nested exception is java.net.ConnectException: Connection timed out: connect
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:785)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:711)
	at org.springframework.web.client.RestTemplate.postForObject(RestTemplate.java:437)
	at ms.apiclient.client.common.ApiRestTemplate.call(ApiRestTemplate.java:25)
	at ms.apiclient.client.common.ApiCallClient.call(ApiCallClient.java:36)
	at ms.apiclient.t24util.T24UtilClient.login(T24UtilClient.java:13)
	at com.kaiasia.app.service.Auth_api.api.login.LoginService.process(LoginService.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.kaiasia.app.register.RequestProcessor.execute(RequestProcessor.java:20)
	at com.kaiasia.app.register.ServiceRegister.processAuthenType(ServiceRegister.java:81)
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:35)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1168)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1104)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:998)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:932)
	at org.springframework.http.client.SimpleBufferingClientHttpRequest.executeInternal(SimpleBufferingClientHttpRequest.java:76)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:776)
	... 19 common frames omitted
07-01-2025 21:43:51.359 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:44460
07-01-2025 21:43:51.361 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c#saveResponse#Duration: 2
07-01-2025 21:43:51.513 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c##END
07-01-2025 21:43:51.513 INFO  --- com.kaiasia.app.core.MainRestController  - 1736260986771000_0000_422a3735b08144a08b3bd225fc2b4f9c#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=44359, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=998, desc=Timeout))#Total time: 44772
07-01-2025 21:47:25.121 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 14808 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:47:25.124 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:47:25.124 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:47:26.873 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:47:27.698 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 2.995 seconds (JVM running for 4.052)
07-01-2025 21:47:40.077 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:47:40.084 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:47:40.103 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:47:40.104 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c#[API-BEGIN]
07-01-2025 21:47:40.131 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c#BEGIN
07-01-2025 21:47:40.138 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c#END
07-01-2025 21:47:40.139 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c##pushApiReqToQueue#Duration:35
07-01-2025 21:47:40.232 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 1
07-01-2025 21:47:40.232 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:47:40.232 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:47:40.232 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:47:46.368 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:47:48.455 ERROR --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#while calling api T24_UTIL_API: I/O error on POST request for "http://localhost:8082/T24_UTIL_API/process": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
07-01-2025 21:47:48.457 ERROR --- ms.apiclient.client.common.ApiCallClient - {}#exception {}
org.springframework.web.client.ResourceAccessException: I/O error on POST request for "http://localhost:8082/T24_UTIL_API/process": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:785)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:711)
	at org.springframework.web.client.RestTemplate.postForObject(RestTemplate.java:437)
	at ms.apiclient.client.common.ApiRestTemplate.call(ApiRestTemplate.java:25)
	at ms.apiclient.client.common.ApiCallClient.call(ApiCallClient.java:36)
	at ms.apiclient.t24util.T24UtilClient.login(T24UtilClient.java:13)
	at com.kaiasia.app.service.Auth_api.api.login.LoginService.process(LoginService.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.kaiasia.app.register.RequestProcessor.execute(RequestProcessor.java:20)
	at com.kaiasia.app.register.ServiceRegister.processAuthenType(ServiceRegister.java:81)
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:35)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1168)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1104)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:998)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:932)
	at org.springframework.http.client.SimpleBufferingClientHttpRequest.executeInternal(SimpleBufferingClientHttpRequest.java:76)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:776)
	... 19 common frames omitted
07-01-2025 21:47:48.479 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:8247
07-01-2025 21:47:48.483 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c#saveResponse#Duration: 4
07-01-2025 21:47:48.614 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c##END
07-01-2025 21:47:48.614 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261260104000_0000_76d0ffd9c75148038e9043c288284b5c#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=8479, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=998, desc=Timeout))#Total time: 8537
07-01-2025 21:49:01.607 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:49:01.608 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:49:01.608 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:49:01.608 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261341608000_0000_e616305ad5214d7583698b578496d8da#[API-BEGIN]
07-01-2025 21:49:01.608 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261341608000_0000_e616305ad5214d7583698b578496d8da#BEGIN
07-01-2025 21:49:01.610 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261341608000_0000_e616305ad5214d7583698b578496d8da#END
07-01-2025 21:49:01.611 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261341608000_0000_e616305ad5214d7583698b578496d8da##pushApiReqToQueue#Duration:3
07-01-2025 21:49:01.893 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:49:01.894 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261341608000_0000_e616305ad5214d7583698b578496d8da, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:49:01.894 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:49:01.894 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:49:08.065 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:49:08.238 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 21:49:36.179 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:34285
07-01-2025 21:49:36.185 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261341608000_0000_e616305ad5214d7583698b578496d8da#saveResponse#Duration: 6
07-01-2025 21:49:36.384 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261341608000_0000_e616305ad5214d7583698b578496d8da##END
07-01-2025 21:49:36.384 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261341608000_0000_e616305ad5214d7583698b578496d8da#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=34179, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=801, desc=null))#Total time: 34777
07-01-2025 21:50:17.053 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 17916 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:50:17.055 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:50:17.055 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:50:18.889 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:50:19.270 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 2.572 seconds (JVM running for 3.559)
07-01-2025 21:50:27.429 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:50:27.434 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:50:27.444 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:50:27.445 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4#[API-BEGIN]
07-01-2025 21:50:27.455 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4#BEGIN
07-01-2025 21:50:27.461 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4#END
07-01-2025 21:50:27.463 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4##pushApiReqToQueue#Duration:18
07-01-2025 21:50:27.893 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4##END
07-01-2025 21:50:27.894 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261427444000_0000_63bbdaa08fb7438a9dfe3bd8b3d9e5c4#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=T24_UTIL_API, apiKey=VDI0X1VUSUxfQVBJ, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=822, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=333, desc=Invalid authenType))#Total time: 465
07-01-2025 21:53:22.109 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 14040 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:53:22.111 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:53:22.111 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:53:24.060 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:53:24.480 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 2.746 seconds (JVM running for 3.725)
07-01-2025 21:53:32.651 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:53:32.656 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:53:32.668 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:53:32.669 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a#[API-BEGIN]
07-01-2025 21:53:32.685 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a#BEGIN
07-01-2025 21:53:32.691 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a#END
07-01-2025 21:53:32.692 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a##pushApiReqToQueue#Duration:23
07-01-2025 21:53:33.327 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:53:33.328 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:53:33.328 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:53:33.328 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:53:33.329 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:53:45.790 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 21:53:57.049 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:23721
07-01-2025 21:53:57.051 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a#saveResponse#Duration: 2
07-01-2025 21:53:57.264 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a##END
07-01-2025 21:53:57.264 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261612669000_0000_d6e4cea8d348403ba4c43020936cfc7a#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=24049, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=04, desc=Sai Password#password))#Total time: 24613
07-01-2025 21:55:38.381 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:55:38.382 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:55:38.382 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:55:38.382 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e#[API-BEGIN]
07-01-2025 21:55:38.383 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e#BEGIN
07-01-2025 21:55:38.384 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e#END
07-01-2025 21:55:38.384 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e##pushApiReqToQueue#Duration:2
07-01-2025 21:55:38.711 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:55:38.712 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:55:38.712 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:55:38.712 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:55:38.712 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:55:39.617 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 21:56:00.552 ERROR --- com.kaiasia.app.register.ServiceRegister - {}
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.kaiasia.app.register.RequestProcessor.execute(RequestProcessor.java:20)
	at com.kaiasia.app.register.ServiceRegister.processAuthenType(ServiceRegister.java:81)
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:35)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException: null
	at com.kaiasia.app.service.Auth_api.api.login.LoginService.process(LoginService.java:98)
	... 13 common frames omitted
07-01-2025 21:56:00.552 ERROR --- com.kaiasia.app.ProcessImpl              - {}
java.lang.NullPointerException: null
	at com.kaiasia.app.ProcessImpl.process(ProcessImpl.java:38)
	at com.kaiasia.app.core.ProcessThread.process(ProcessThread.java:58)
	at com.kaiasia.app.core.job.ApiResTask.execute(ApiResTask.java:25)
	at com.kaiasia.app.core.async.KaiTask.run(KaiTask.java:15)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
07-01-2025 21:56:00.555 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e#saveResponse#Duration: 3
07-01-2025 21:56:00.758 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e##END
07-01-2025 21:56:00.758 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261738382000_0000_51d5a8c392c142bea6026a22bb26e93e#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=22552, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=999, desc=Internal Server Error java.lang.NullPointerException))#Total time: 22377
07-01-2025 21:58:45.098 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 16576 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 21:58:45.101 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 21:58:45.101 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 21:58:46.928 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 21:58:47.354 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 2.633 seconds (JVM running for 3.616)
07-01-2025 21:58:50.365 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:58:50.370 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:58:50.384 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:58:50.384 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261930384000_0000_f0c464f32b804bf5972c3677182f667e#[API-BEGIN]
07-01-2025 21:58:50.398 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261930384000_0000_f0c464f32b804bf5972c3677182f667e#BEGIN
07-01-2025 21:58:50.404 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261930384000_0000_f0c464f32b804bf5972c3677182f667e#END
07-01-2025 21:58:50.406 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261930384000_0000_f0c464f32b804bf5972c3677182f667e##pushApiReqToQueue#Duration:22
07-01-2025 21:58:50.565 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:58:50.566 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261930384000_0000_f0c464f32b804bf5972c3677182f667e, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:58:50.566 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:58:50.566 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:58:50.567 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:58:51.073 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 21:59:41.595 ERROR --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null:null
07-01-2025 21:59:41.596 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:51030
07-01-2025 21:59:41.599 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261930384000_0000_f0c464f32b804bf5972c3677182f667e#saveResponse#Duration: 3
07-01-2025 21:59:41.813 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261930384000_0000_f0c464f32b804bf5972c3677182f667e##END
07-01-2025 21:59:41.813 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261930384000_0000_f0c464f32b804bf5972c3677182f667e#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=51596, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=999, desc=Internal Server Error null))#Total time: 51448
07-01-2025 21:59:45.057 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:59:45.057 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 21:59:45.057 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 21:59:45.057 INFO  --- com.kaiasia.app.core.MainRestController  - 1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47#[API-BEGIN]
07-01-2025 21:59:45.058 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47#BEGIN
07-01-2025 21:59:45.059 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47#END
07-01-2025 21:59:45.062 INFO  --- com.kaiasia.app.core.ProcessService      - 1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47##pushApiReqToQueue#Duration:5
07-01-2025 21:59:45.676 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 21:59:45.676 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 21:59:45.676 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 21:59:45.676 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 21:59:45.676 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 21:59:46.129 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 22:00:08.433 ERROR --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null:null
07-01-2025 22:00:08.435 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:22759
07-01-2025 22:00:08.436 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736261985057000_0000_fdbf405b6e764e05bc447d3a5e316b47#saveResponse#Duration: 1
07-01-2025 22:00:18.437 INFO  --- com.kaiasia.app.AppApplication           - Starting AppApplication using Java 1.8.0_45 on NguyenLoc with PID 4424 (C:\Users\Admin\Desktop\KAI_ASIA\Auth_api\target\classes started by Admin in C:\Users\Admin\Desktop\KAI_ASIA\Auth_api)
07-01-2025 22:00:18.440 DEBUG --- com.kaiasia.app.AppApplication           - Running with Spring Boot v2.7.2, Spring v5.3.22
07-01-2025 22:00:18.441 INFO  --- com.kaiasia.app.AppApplication           - No active profile set, falling back to 1 default profile: "default"
07-01-2025 22:00:20.456 DEBUG --- com.kaiasia.app.register.ServiceRegister - Found 3 classes with annotation SeabService
07-01-2025 22:00:20.848 INFO  --- com.kaiasia.app.AppApplication           - Started AppApplication in 2.801 seconds (JVM running for 3.908)
07-01-2025 22:00:22.924 INFO  --- com.kaiasia.app.core.MainRestController  - MsgFromClient_Receive:ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=null, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 22:00:22.928 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method validate for AuthenType: VALIDATE_login
07-01-2025 22:00:22.942 INFO  --- com.kaiasia.app.ValidateImpl             - 000 OK
07-01-2025 22:00:22.942 INFO  --- com.kaiasia.app.core.MainRestController  - 1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321#[API-BEGIN]
07-01-2025 22:00:22.957 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321#BEGIN
07-01-2025 22:00:22.962 INFO  --- com.kaiasia.app.core.dao.APIDaoManager   - InsertAPIReqs()1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321#END
07-01-2025 22:00:22.963 INFO  --- com.kaiasia.app.core.ProcessService      - 1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321##pushApiReqToQueue#Duration:21
07-01-2025 22:00:23.095 INFO  --- com.kaiasia.app.core.ProcessThread       - checkTimeInQueue: 0
07-01-2025 22:00:23.095 DEBUG --- com.kaiasia.app.ProcessImpl              - received msg: ApiRequest(header=ApiHeader(reqType=REQUEST, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=null, priority=1, context=null, synasyn=false), body={command=GET_ENQUIRY, enquiry={authenType=login, username=28169d2, password=password}, status=OK}, _reqId=1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321, _rejectCounter=null, _sendTime=null, _command=null, _authenType=null, _timeOut=null)
07-01-2025 22:00:23.095 DEBUG --- com.kaiasia.app.register.ServiceRegister - Execute method AuthenType: login
07-01-2025 22:00:23.097 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#BEGIN
07-01-2025 22:00:23.099 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#begin call api T24_UTIL_API
07-01-2025 22:00:23.382 INFO  --- ms.apiclient.client.common.ApiCallClient - API-28169d2-null#end call api T24_UTIL_API
07-01-2025 22:00:38.693 ERROR --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null:PreparedStatementCallback; bad SQL grammar [INSERT INTO auth_api.auth_session
(username, start_time, end_time, session_id, channel, "location", phone, email, company_code, customer_id)
VALUES(?, ?, ?,?, ?, ?,?, ?, ?, ?);]; nested exception is org.postgresql.util.PSQLException: ERROR: relation "auth_api.auth_session" does not exist
  Position: 13
07-01-2025 22:00:38.693 INFO  --- com.kaiasia.app.service.Auth_api.api.login.LoginService - API-28169d2-null#END#Duration:15596
07-01-2025 22:00:38.700 INFO  --- com.kaiasia.app.core.ProcessThread       - 1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321#saveResponse#Duration: 6
07-01-2025 22:00:38.912 INFO  --- com.kaiasia.app.core.MainRestController  - 1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321##END
07-01-2025 22:00:38.912 INFO  --- com.kaiasia.app.core.MainRestController  - 1736262022942000_0000_49f7a30417c94b59bdf13b65e87e1321#MsgFromClient_Response:ApiResponse(header=ApiHeader(reqType=RESPONSE, api=Auth_API, apiKey=authq51klfoni1ezxl5f2ckpfx248, channel=API, location=PC/IOS, requestAPI=FE API, requestNode=node 01, duration=15694, priority=1, context=null, synasyn=false), body={status=FAILE}, error=ApiError(code=999, desc=Internal Server Error PreparedStatementCallback; bad SQL grammar [INSERT INTO auth_api.auth_session
(username, start_time, end_time, session_id, channel, "location", phone, email, company_code, customer_id)
VALUES(?, ?, ?,?, ?, ?,?, ?, ?, ?);]; nested exception is org.postgresql.util.PSQLException: ERROR: relation "auth_api.auth_session" does not exist
  Position: 13))#Total time: 15989
